BrainPortal/app/controllers/users_controller.rb

```diff
@@ -328,6 +329,31 @@ def destroy #:nodoc:
     end
   end
 
+  # Only use via API and accessible only to admin users
```

Comment:
I would rephrase in polite  "Please, use only via API ..." 
or non-prescriptive manner "API-only action ..."